src/main/native/cpp/photonlib/SimVisionSystem.cpp

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (C) 2020-2021 Photon Vision.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "photonlib/SimVisionSystem.h"
+
+#include <cmath>
+
+namespace photonlib {
+
+SimVisionSystem::SimVisionSystem(
+    const std::string& name, units::angle::degree_t camDiagFOVDegrees,
+    units::angle::degree_t camPitchDegrees, frc::Transform2d robotToCamera,
+    units::length::meter_t cameraHeightOffGroundMeters,
+    units::length::meter_t maxLEDRangeMeters, int cameraResWidth,
```

Comment:
You don't need to append the parameter names with the units -- that is only necessary in Java.